### PR TITLE
Show Basic System info

### PR DESCRIPTION
**Pull request by Shosoul**
This will help people see some stats. It will also allow them to find the path to the save folder.

### CODE REVIEW

File: options.lua [hunk 1]
```diff
@@ -593,6 +593,47 @@ local function about_menu(button_idx)
...
+        sys_info[#sys_info + 1] = {name = "Characters [Enabled/Total]", value = #characters_ids_for_current_theme.."/"..#characters_ids} 
+        sys_info[#sys_info + 1] = {name = "Stages [Enabled/Total]", value = #stages_ids_for_current_theme.."/"..#stages_ids} 
+        sys_info[#sys_info + 1] = {name = "Total Panel Sets", value = #panels_ids} 
+        sys_info[#sys_info + 1] = {name = "Total Themes", value = #found_themes}
+        local info_string = ""
+        for index, info in ipairs(sys_info) do
+          info_string = info_string .. info.name .. ": " .. (info.value or "Unknown") .. "\n"
+        end
+        while true do
+          gprint(info_string, 0, 15)
```
Comment (JamesVanBoxtel) [inline options.lua:619]: Maybe print a little lower and to the right so its not so squished

File: mainloop.lua [hunk 2]
```diff
@@ -192,11 +192,8 @@ do
         end
       end
       
-      local loveString = Game.loveVersionString()
-      gprintf("Love Version: " .. loveString, -5, 705, canvas_width, "right")
```
Comment (JamesVanBoxtel) [inline mainloop.lua:196]: I'd like to leave the love version showing until we fully move everyone over to 11.4. Maybe only show it with a suggestion to move if they are still on 11.3

File: options.lua [hunk 3]
```diff
@@ -593,6 +593,47 @@ local function about_menu(button_idx)
...
+          wait()
+          local panels_ret = nil
+          variable_step(
+            function()
+              if menu_escape() or menu_enter() then
+                panels_ret = {about_menu, {4}}
```
Comment (JamesVanBoxtel) [inline options.lua:625]:
```suggestion
                panels_ret = {about_menu, {5}}
```
In order to return back to the right spot

File: options.lua (hunk 1 above)
Comment (Shosoul) [reply options.lua]: Fixed

File: mainloop.lua (hunk 2 above)
Comment (Shosoul) [reply mainloop.lua]: Reverted!

**Comment by Shosoul** [comment]
Ok this should be good to go now

**Review by JamesVanBoxtel — APPROVED** [review]
Tested, works well

**Pull request merged**